arviz/plots/ppcplot.py

```diff
@@ -19,7 +19,7 @@ def plot_ppc(
     alpha=None,
     mean=True,
     observed=True,
-    color="C0",
+    colors=None,
```

Comment:
can we keep `color` for a bit and deprecate it so users know they need to change from singular to plural? I expect this change to be quite confusing otherwise.